CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Added
+
+- Add argument to `ci-test-route` to configure the maximum number of retries
+
```

Comment:
As it is related to the project, I'm not sure we should log this... WDYT @lunika?

Reply:
I think we can keep this new entry in the changelog.

Reply:
Thanks ! I'll merge this one when the CI checks are finished then.